fastlane/report.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this final under version control?

Reply:
Nope, will remove